### PR TITLE
Remove line that caused tests to not indent at all in the tests and fix regression

### LINKED ISSUE #64: Some commits caused bad indentation and broke the tests
So if I checkout bb602e1 the tests for indentation breaks badly because it indents things badly, which is great and all that we now have tests. We just need to get feedback with travis (which was being worked on) to get a flow of this.

But what's worse is that 4ac7b5a seems to break the indentation that the tests does so nothing happens at all.

My plan for the moment is to add a test which contains broken indentation so we can have a test that that `(should-not)` match the input file to detect this kind of broken.

I'm not sure what bb602e1 was supposed to do overall since it touched many places. But I find some things that I would consider regressions straight of by reading that commit.

One example, `nix-is-comment-p` was added which is great and all and get the 3rd element from the result of `syntax-ppss`:
https://github.com/NixOS/nix-mode/commit/bb602e160fa5bb8ae15b113ebf308f0fb30b3aa9#diff-e75a5e47fe55dc4b6e12cf1e4c7a42adR521

And that function is then used here:
https://github.com/NixOS/nix-mode/commit/bb602e160fa5bb8ae15b113ebf308f0fb30b3aa9#diff-e75a5e47fe55dc4b6e12cf1e4c7a42adL528

Where the original code read the 3th element from the result of `syntax-ppss`.

Changing 3 to 4 there didn't repair the tests though.

@matthewbauer

### PR DESCRIPTION
**Pull request by etu**
This fixes #64.

Still work in progress. Just wanted to see the test failure from travis.

### CODE REVIEW

**Comment by etu** [comment]
So got a test failure on the first commit, which was the point. And that failure should be fixed with the second commit.

File: nix-mode.el [hunk 1]
```diff
@@ -117,7 +117,7 @@ Valid functions for this are:
 
 (defconst nix-re-quotes "''\\|\"")
 
-(defconst nix-re-comments "#\\|/\\*\\|\\*/")
+(defconst nix-re-comments "#\\|/*\\|*/")
```
Comment (matthewbauer) [inline nix-mode.el]: I think the second ```\\``` should be left in here

File: nix-mode.el [hunk 2]
```diff
@@ -628,7 +628,6 @@ The hook `nix-mode-hook' is run when Nix mode is started.
 
   ;; Automatic indentation [C-j]
   (setq-local indent-line-function nix-indent-function)
-  (set (make-local-variable 'indent-region-function) 'nix-indent-region)
```
Comment (matthewbauer) [inline nix-mode.el]: Was this line causing issues?

File: nix-mode.el (hunk 1 above)
Comment (matthewbauer) [inline nix-mode.el]: Hm... Perhaps I don't know emacs regex as well as I thought. But shouldn't the ```*``` be escaped here? You want it to match ```/*``` and ```*/```.

Comment (etu) [reply nix-mode.el]: Well, my elisp regex is bad as well. I just evalled the older revision of the file and grabbed the value from there. So the one I changed it to here is the one that been in use for ages.

**Pull request closed without merging**